packages/ffe-account-selector-react/src/components/account-selector/AccountSelector.js

```diff
@@ -60,24 +65,34 @@ const AccountSelector = ({
         }
     };
 
+    const onBlur = () => {
+        if (allowCustomAccount) {
+            onAccountSelected({
+                name: inputValue,
+                accountNumber: inputValue,
+            });
+        }
+        if (inputProps && inputProps.onBlur) {
```

Comment:
if(typeof inputProps?.onBlur === 'function') ?

Reply:
Er det ikke derfor vi har typescript? :trollface:

Reply:
Å sjekke at `onBlur` er en funksjon blir litt for mye defensivt. Hvis man sender innen noe annet enn en funksjon i en prop som heter `onBlur` så har man bommet skikkelig og har fortjent å debugge en «x is not a function».

Reply:
Jeg tenker det kanskje holder å endre til `if (inputProps?.onBlur) {` :blush:

Reply:
Greit for meg :)